### PR TITLE
Add script to run games more easily

### PR DESCRIPTION
**Pull request by aphedges**
I created a script to make running games on both locally and on TACC much easier. It can only run one type of agent with one configuration, but it should make general testing much easier.

I included two other commits that contain some relevant changes.

### CODE REVIEW

File: scripts/create_game.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    rules: Sequence[str] = DEFAULT_RULES,
+    deadline: int = 0,
+    n_controls: int = 7,
+    password: Optional[str] = None,
+    hostname: str = "localhost",
+    port: int = 8432,
+) -> None:
+    """Creates a game on the Diplomacy server"""
+    connection = await connect(hostname, port)
+    channel = await connection.authenticate("random_user", "password")
```
Comment (DenisPeskov) [inline scripts/create_game.py:26]: Random user might be best to make a meaningful name if we submit a game for TACC review

File: containers/allan_dip_bot/BUILD.md [hunk 2]
```diff
@@ -22,6 +22,6 @@ $ docker run -it allan_dip_bot --help
 --outdir    OUT_DIR
 
 #connect to remote game engine
-$ docker run -it allan_dip_bot --game_id test_game --host shade.tacc.utexas.edu --power TURKEY
+$ docker run -it allan_dip_bot --game_id test_game --host shade-dev.tacc.utexas.edu --power TURKEY
```
Comment (DenisPeskov) [inline containers/allan_dip_bot/BUILD.md:25]: Not in this PR but why is this hard coded to Turkey?

File: tests/bot_tests/SOA_test.py [hunk 3]
```diff
@@ -62,7 +62,7 @@ def test_play(self):
 
     @testing.gen_test
     def test_send_message(self):
-        hostname = "shade.tacc.utexas.edu"
+        hostname = "shade-dev.tacc.utexas.edu"
```
Comment (DenisPeskov) [inline tests/bot_tests/SOA_test.py:65]: Might make sense to add flag for dev vs main tacc down the road.  Fine for this and not sure of significance between the two.

File: containers/allan_dip_bot/BUILD.md (hunk 2 above)
Comment (aphedges) [reply containers/allan_dip_bot/BUILD.md:25]: This is just an example command in a documentation file.

File: scripts/create_game.py (hunk 1 above)
Comment (aphedges) [reply scripts/create_game.py:26]: I agree. After the problem today, I made a note to have a better default for this.

**Review by DenisPeskov — APPROVED** [review]
Seems reasonable on skim.  May need more detailed script review if more people use it.  Small comments added throughout.

File: tests/bot_tests/SOA_test.py (hunk 3 above)
Comment (aphedges) [reply tests/bot_tests/SOA_test.py:65]: I have no clue. This is the only spot it's referenced in the main code, and changing it didn't affect anything.

**Pull request merged**